react/components/EXPERIMENTAL_Table/DataTable/NodeRow.tsx

```diff
@@ -67,16 +68,7 @@ const Row: FC<RowProps> = ({ data, index, depth }) => {
               id={`${index}-${cellIndex}-${depth}`}
               key={`cel-${index}-${cellIndex}-${depth}`}
               width={width}>
-              {nestedRows && cellIndex === 0 && (
-                <Cell.Prefix width={prefixWidth}>
-                  {arrow && (
-                    <Cell.Prefix.Arrow
-                      active={collapsed}
-                      onClick={() => setCollapsed(!collapsed)}
-                    />
-                  )}
-                </Cell.Prefix>
-              )}
+              {renderPrefix(cellIndex === 0, hasChild)}
```

Comment:
What about this instead of putting this logic inside the component?

```suggestion
              {cellIndex > 0 && renderPrefix(hasChild)}
```

Reply:
Great, nice tip